web/hooks/use-has-created-contract-today.ts

```diff
@@ -53,7 +54,14 @@ export const useHasCreatedContractToday = (user: User | null | undefined) => {
       setHasCreatedContractToday(sessionCreatedContractToday)
```

Comment:
What's the purpose of `sessionCreatedContractToday`? Seems like it's not read other than here.